peer/peerstore.go

```diff
@@ -0,0 +1,153 @@
+package peer
```

Comment:
This whole file contains all the code that used to be in peer.go that involved the PeerStore. No need to review it in detail, I didn't change any of the functionality.